pkg/providers/amifamily/bootstrap/bootstrap.go

```diff
@@ -78,12 +78,9 @@ func (o Options) kubeletExtraArgs() (args []string) {
 }
 
 func (o Options) nodeTaintArg() string {
-	if len(o.Taints) == 0 {
-		return ""
-	}
 	var taintStrings []string
 	for _, taint := range o.Taints {
-		taintStrings = append(taintStrings, fmt.Sprintf("%s=%s:%s", taint.Key, taint.Value, taint.Effect))
+		taintStrings = append(taintStrings, fmt.Sprintf("%s%s:%s", taint.Key, lo.Ternary(taint.Value == "", "", "="+taint.Value), taint.Effect))
```

Comment:
Can we actually just use `taint.ToString()` here?

Reply:
Yep, nice catch.